docs/DeveloperGuide.md

```diff
@@ -552,21 +552,99 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
-### Deleting a person
+### Command Testing
+#### Adding A Person or A Company
+1. Adding a person
 
-1. Deleting a person while all persons are being shown
+    1. Test case: `add p n/Tom p/12345678 e/tom@gmail.com a/Kent Ridge Road t/friend`<br>
+       Expected: A person contact with the given information is added to the end of the person contact list. 
+            Details of the newly added person contact shown in the status message. Timestamp in the status bar is updated.
 
-   1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
+    1. Test case: `add p n/Tom`<br>
+       Expected: No person is added due to field missing(Only the tag field with `t/` prefix is optional). Error details shown in the status message. Status bar remains the same.
```

Comment:
Missing space between 'missing' and '(Only...'. Will merge and correct it on my end in the interest of time

Reply:
Fixed in the new commit.